### PR TITLE
Migrate Kredivo to Invoice

### PR DESCRIPTION
**Pull request by IreneGohtami**
## Description

- Migrate Kredivo to Invoice
- Cleanup unused cardless credit + ewallets functions & files
- Rename Cardless credit label -> PayLater
- Fix naming typos

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot
Screenshot 2021-11-02 at 11 32 28 AM

### CODE REVIEW

File: Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php [hunk 1]
```diff
@@ -26,8 +26,10 @@ public function execute()
             $transactionAmount  = 0;
             $orderProcessed     = false;
             $orders             = [];
+            $addresses          = [];
+            $items              = [];
 
-            $orderIncrementIds = '';
+            $orderIncrementIds  = '';
 
             $c = 0;
```
Comment (brijal1987) [inline Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php:34]: Give proper variable name here ..

Comment (rendegosling) [reply Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php:34]: seems old code, agree that `c` and `0` doesn't give context what it represents

**Review by rendegosling — APPROVED** [review]
thats a lot of deleted code. LGTM

File: Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php (hunk 1 above)
Comment (IreneGohtami) [reply Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php:34]: $c stands for count.. but it's used in other files as well, so let's leave it be.

**Review by yanoandri — APPROVED** [review]

**Review by kevinalfianto — APPROVED** [review]

**Pull request merged**